compiler/compile/OSRData.hpp

```diff
@@ -119,7 +119,7 @@ class TR_OSRCompilationData
                            int32_t symRefOrder, int32_t symSize, bool takesTwoSlots);
    void ensureSlotSharingInfoAt(const TR_ByteCodeInfo& T);
    void addInstruction(TR::Instruction* instr);
-   void addInstruction(int32_t instructionPC, TR_ByteCodeInfo &bcInfo);
+   void addInstruction(int32_t instructionPC, TR_ByteCodeInfo bcInfo);
```

Comment:
Okay I did not get that the value was actually being changed (at line 139) until your response to my question. I agree with the change in that case since the caller side value should not be changed just because of the iteration over the call stack in this routine.